pkg/build/generator/generator.go

```diff
@@ -218,6 +218,10 @@ func (g *BuildGenerator) Instantiate(ctx kapi.Context, request *buildapi.BuildRe
 		return nil, err
 	}
 
+	if bc.Spec.Strategy.ExternalStrategy != nil {
+		return nil, &GeneratorFatalError{fmt.Sprintf("can't instantiate from BuildConfig %s/%s: BuildConfig uses an External build strategy", bc.Namespace, bc.Name)}
+	}
```

Comment:
(that will necessitate changes to our build controller logic to ignore these though.)